app/components/draft_scheduled_post/draft_scheduled_post.tsx

```diff
@@ -83,7 +83,12 @@ const DraftAndScheduledPost: React.FC<Props> = ({
 
     const onLongPress = useCallback(() => {
         Keyboard.dismiss();
-        const title = isTablet ? intl.formatMessage({id: 'draft.options.title', defaultMessage: 'Draft Options'}) : 'Draft Options';
+        let title;
+        if (draftType === DRAFT_TYPE_DRAFT) {
+            title = isTablet ? intl.formatMessage({id: 'draft.options.title', defaultMessage: 'Draft Options'}) : 'Draft Options';
+        } else {
+            title = isTablet ? intl.formatMessage({id: 'scheduled_post.options.title', defaultMessage: 'Message actions'}) : 'Message actions';
```

Comment:
Explain this please.. why the diff between tablet and not.. just keep intl version even if not displayed on phones

Reply:
Ahh, true. Nice catch @enahum.

Reply:
Done.